orderer/consensus/hlmirbft/chain.go

```diff
@@ -523,6 +530,53 @@ func (c *Chain) Consensus(req *orderer.ConsensusRequest, sender uint64) error {
 
 	return nil
 }
+//JIRA FLY2-64 : Proposed Change:New function to propose normal messages to node
+func (c *Chain) proposeMsg(msg *orderer.SubmitRequest) (err error) {
+
+	//JIRA FLY2-64 proposed changes: added standerdized code for dealing with requests.
+    //For Testing, we have derived the clientID and request number using the following code convert common.envelope into signedData list.
```

Comment:
It should be safe to reuse MirBFTID as the clientID since these IDs only have to be unique relative to the other IDs in their respective set. See how I've initialised the mirbft network state in FLY2-55. Each node will then own the client which has the same ID.

Reply:
This one fixed.

Reply:
Would you mind splitting up your pull requests so that they only include the code associated with the ticket? e.g. FLY2-55 contains only code added for the FLY2-55 ticket. It be great if you could since it makes reviewing a lot easier. In future you can checkout the main branch before branching a JIRA ticket to avoid including code added in a different branch.